tests/web3js/eth_streaming_test.js

```diff
@@ -26,56 +26,63 @@ it('streaming of logs using filters', async() => {
     await new Promise((res, rej) => setTimeout(() => res(), 1000))
 
     // subscribe to new blocks being produced by bellow transaction submission
-    let blockCount = 0
-    let blockHashes = []
+    let blockTxHashes = []
+    let subBlocks = await ws.eth.subscribe('newBlockHeaders')
     let doneBlocks = new Promise(async (res, rej) => {
-        let subBlocks = await ws.eth.subscribe('newBlockHeaders')
+        subBlocks.on("error", err => {
+            rej(err)
+        })
+
         subBlocks.on('data', async block => {
-            blockHashes.push(block.transactions[0]) // add received tx hash
+            blockTxHashes.push(block.transactions[0]) // add received tx hash
 
-            blockCount++
-            if (blockCount === testValues.length) {
-                await subBlocks.unsubscribe()
+            if (blockTxHashes.length === testValues.length) {
+                subBlocks.unsubscribe()
```

Comment:
Removed the `await` from

```javascript
await subBlocks.unsubscribe()
```

as this might be causing the blocking time to increase, hence reaching the timeout limit.

Reply:
the problem with this I see, is that we now ignore any issues with unsubscribing, which would be better if test fails if there are issues. Unsubscribe shouldn't take long, so I rather increase the time a bit than not wait for result

Reply:
I took this pattern from `eth_streaming_filters_test`, which has never failed in CI. If we observe any problem, we can bring it back.